.github/workflows/ci.yml

```
@@ -225,8 +225,8 @@ jobs:
           TYPESCRIPT_ESLINT_EXPERIMENTAL_TSSERVER: true
 
   website_tests:
-    # The NETLIFY_TOKEN secret will not be available on forks
-    if: github.repository_owner == 'typescript-eslint'
+    # The NETLIFY_TOKEN secret will not be available on PRs from forks
+    if: (github.event_name == 'push' && github.ref_name == 'main') || (github.event.pull_request.head.repo.full_name == github.repository)
```

This change does not block PRs in a fork to the same fork.
If we really need to skip all PRs in forks, this may be following 

```suggestion
    # The NETLIFY_TOKEN secret will not be available on forks
    if: (github.event_name == 'push' && github.ref_name == 'main') || (github.event.pull_request.head.repo.full_name == 'typescript-eslint/typescript-eslint')
```